src/Compilers/Test/Resources/Core/NetFX/Minimal/mincorlib.cs

```diff
@@ -2,8 +2,8 @@
 
 namespace System
 {
-	public class Object {}
-    public struct Byte {}
+    public class Object { }
```

Comment:
Aren't all special types and members required to be in a valid corlib? I believe so and if that's true then any existing tests were testing the wrong thing.